community.md

```diff
@@ -71,6 +71,7 @@ It would be great to hear from you especially if you have any of the above and w
 
 | Title                                                              | Presenter       | Date      |
 | ------------------------------------------------------------------ | --------------- | --------- |
```

Comment:
I think I would put these into the blog posts section instead of the featured / official videos?

Reply:
I have updated heading a little bit.